### PR TITLE
fix a typo

### PR DESCRIPTION
**Pull request by Son-Dongwoo**
The word "becaose" seems a typo. This PR corrects the typo to "because".

### CODE REVIEW

**Review by sunmy2019 — APPROVED** [review]

**Comment by serhiy-storchaka** [comment]
There is another typo in this line.

File: Tools/c-analyzer/c_parser/info.py [hunk 1]
```diff
@@ -206,7 +206,7 @@ def from_row(cls, row, **markers):
         row = _tables.fix_row(row, **markers)
         return cls(*row)
 
-    # We have to provde _make() becaose we implemented __new__().
+    # We have to provde _make() because we implemented __new__().
```
Comment (dcollison) [inline Tools/c-analyzer/c_parser/info.py]:
```suggestion
    # We have to provide _make() because we implemented __new__().
```

**Review by ericsnowcurrently — APPROVED** [review]
LGTM (with @dcollison's recommended fix)

**Comment by ericsnowcurrently** [comment]
(I *really* need to use a spell-checker! 😆)

**Comment by sunmy2019** [comment]
@ericsnowcurrently Can you land it?

**Pull request merged**